src/applications/facility-locator/components/ProviderServiceDescription.jsx

```diff
@@ -14,7 +14,7 @@ import React from 'react';
 const ProviderServiceDescription = ({ provider, details = false }) => {
   if (details) {
     const { specialty } = provider.attributes;
-    if (specialty.length < 1) return null;
+    if (specialty && specialty.length < 1) return null;
```

Comment:
I added this additional check per a discussion with Rian. I was seeing an error on localhost community provider detail views, sometimes. Error log is attached as a text file for reference.

---
[provider-detail-specialty-log.txt](https://github.com/department-of-veterans-affairs/vets-website/files/3315033/provider-detail-specialty-log.txt)